test/runtests.py

```diff
@@ -535,19 +544,22 @@ def main():
         args.folders = [f for f in sorted(files) if not os.path.isfile(f)]
 
     # Set the right timezone, the tests need Pacific Standard Time
-    os.environ['TZ'] = 'US/Pacific'
-    time.tzset()
+    if hasattr(time, 'tzset'):  # only available on Unix
```

Comment:
Can you please add a 'todo' note here? I think we can have a workaround for Windows here.